CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+For more information, see https://docs.dask.org/en/latest/develop.html#contributing-to-code
+
+
+## Style
+Distributed conforms with the [flake8] and [black] styles. To make sure your
+code conforms with these styles, run
+
+``` shell
+$ pip install black flake8
+$ cd path/to/distributed
+$ black .
```

Comment:
```suggestion
$ black distributed
```

Otherwise, I think that this also applies to code like setup.py and versioneer.

Reply:
Ah, got it. I've resolved both these changes.